src/components/rangeslider/defaults.js

```diff
@@ -38,6 +44,34 @@ module.exports = function handleDefaults(layoutIn, layoutOut, axName) {
     coerce('autorange', !axOut.isValidRange(containerIn.range));
     coerce('range');
 
+    var subplots = layoutOut._subplots;
+    if(subplots) {
+        var yIds = subplots.cartesian
+            .filter(function(subplotId) {
+                return subplotId.substr(0, subplotId.indexOf('y')) === axisIds.name2id(axName);
+            })
+            .map(function(subplotId) {
+                return subplotId.substr(subplotId.indexOf('y'), subplotId.length);
+            });
+        var yNames = Lib.simpleMap(yIds, axisIds.id2name);
+        for(var i = 0; i < yNames.length; i++) {
+            var yName = yNames[i];
+
+            var rangeContainerIn = containerIn[yName] || {};
+            var rangeContainerOut = containerOut[yName] = {};
+
+            if(rangeContainerIn.range && layoutOut[yName].isValidRange(rangeContainerIn.range)) {
```

Comment:
Hmm. The default `rangemode` value should be `'match'` to match (:laughing: ) the current behavior. We could change that in `v2`, but in the meantime we'll have to make sure this PR doesn't induce a breaking change,

Reply:
Done.